integration/aws_auth.go

```diff
@@ -0,0 +1,20 @@
+package integration
+
+import (
+	"encoding/json"
+	"os"
+)
+
+type awsCredentialsFile struct {
+	RoleArn    string `json:"role_arn"`
+	ExternalID string `json:"external_id"`
+}
+
+func awsLoadDefaultCredentials() (awsCredentialsFile, error) {
+	return awsLoadCredentials("AWS_CREDS")
```

Comment:
I am assuming you documented these credentials in our confluence page, right?

Reply:
Yes, under 'AwsCfg Integration Test'